src/main.cpp

```diff
@@ -403,10 +403,10 @@ void handle_graceful_termination(int)
     exit_main_loop = true;
 }
 
-static bool user_specified_overwrite(std::string filename)
+static bool user_specified_overwrite(std::string filename, std::string muxername)
 {
     struct stat buffer;   
-    if (stat (filename.c_str(), &buffer) == 0)
+    if (stat (filename.c_str(), &buffer) == 0 && muxername != "v4l2")
```

Comment:
I'm thinking, can we not just check whether the file is a character device? Why do we need to special case v4l2?